<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialConnectionWidget</class>
 <widget class="QWidget" name="SerialConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Дополнительные настройки</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Скорость:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="advancedCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Дополнительные настройки</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comPortComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="updatePortsButton">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="serialSpeedComboBox">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comPortComboBox</tabstop>
  <tabstop>updatePortsButton</tabstop>
  <tabstop>serialSpeedComboBox</tabstop>
  <tabstop>advancedCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>advancedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
